include/aspect/simulator.h

```diff
@@ -1193,6 +1193,25 @@ namespace aspect
                                         LinearAlgebra::BlockVector &relevant_dst,
                                         LinearAlgebra::BlockVector &tmp_distributed_stokes);
 
+      /**
+       * Offset the boundary id of all faces located on an outflow bounday
+       * by a fixed value given by the input parameter
+       * @param boundary_id_offset.
```

Comment:
I dislike that this is a parameter. That's something that will be completely bizarre to basically every user who does not look up the code.

I'd much rather than you simply disallow a certain range of boundary indicators (and add a check somewhere) and then do the shift internally.

Reply:
This is how I had it in the beginning, but I need the offset in two functions (`replace_outflow_boundary_ids` and `reset_outflow_boundary_ids`), and it has to be the same in both. This way I at least make sure that the same number is used in both places. Do you have a good suggestion on how to do this?

Reply:
I think you already hard-coded it. That's ok. I know how to fix it later on.